maintainers/team-list.nix

```diff
@@ -858,6 +858,18 @@ with lib.maintainers;
     shortName = "Vim/Neovim";
   };
 
+  nextcloud = {
+    members = [
+      bachp
+      britter
+      dotlambda
```

Comment:
@dotlambda in my head you've been a maintainer and I just got reminded that you aren't. Given you're doing quite a lot of stuff, I figured I could add you. If that's not OK, let me know and I'll remove you again!